tests/connect/test_parquet.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import os
+import shutil
+import tempfile
+
+
+def test_write_parquet(spark_session):
+    # Create a temporary directory
+    temp_dir = tempfile.mkdtemp()
```

Comment:
You can just use `with tempfile.TemporaryDirectory() as dirname:` then you don't need to implement cleanup yourself.